staging/src/k8s.io/kube-apiextensions-server/pkg/apiserver/apiserver.go

```diff
@@ -130,7 +134,18 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 
 	crdClient, err := internalclientset.NewForConfig(s.GenericAPIServer.LoopbackClientConfig)
 	if err != nil {
-		return nil, err
+		// it's really bad that this is leaking here, but until we can fix the test (which I'm pretty sure isn't even testing what it wants to test),
```

Comment:
kind of.  Before when you set that it would fail differently.  Now its on a critical path.  That entire env var needs rethinking, but its bigger than any modification we'd do in 1.7.